src/wp-includes/functions.wp-scripts.php

```diff
@@ -144,7 +145,7 @@ function wp_add_inline_script( $handle, $data, $position = 'after' ) {
 		$data = trim( preg_replace( '#<script[^>]*>(.*)</script>#is', '$1', $data ) );
 	}
 
-	return wp_scripts()->add_inline_script( $handle, $data, $position );
+	return wp_scripts()->add_inline_script( $handle, $data, $position, $standalone );
```

Comment:
I found 47 places in core where we call `wp_add_inline_script`, maybe we should update all of those with the `$standalone` parameter?

Reply:
Just to add, I think this is a good task for us to split out as a follow-up once the initial PR is open, similar to how we might consider adding async/defer to some scripts that core registers by default.